libs/core/src/components/sage-input/stories/sage-input.stories.mdx

```diff
@@ -0,0 +1,119 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+import { html, render } from 'lit-html';
+
+import { extractArgTypes } from '@pxtrn/storybook-addon-docs-stencil';
+
+<Meta
+  title="webcomponents/Input"
+  component="sage-input"
+  argTypes={extractArgTypes('sage-input')}
+  args={{
+    disabled: false,
+    errorText: null,
+    hint: null,
+    name: null,
+    readonly: false,
+    type: 'text'
+}}
+/>
+
+export const Template = (args) => html`<sage-input
+  disabled="${args.disabled}"
+  error-text="${args.errorText}"
+  hint="${args.hint}"
+  id="${args.inputId}"
+  label="${args.label}"
+  name="${args.name}"
+  readonly="${args.readonly}"
+  type="${args.type}"
+  value="${args.value}">
+</sage-input>`;
+
+# Input
+
+<p>
+  A input field coupled with a label that allows a user to type into.
```

Comment:
Wording here feels funny to me. Perhaps something like "An input field coupled with a label and error text which allows for user input and feedback"?